### PR TITLE
Add blockchain_follower

### PR DESCRIPTION
**Pull request by madninja**

### CODE REVIEW

File: src/blockchain_follower.erl [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    {noreply, State}.
+
+handle_info({blockchain_event, From, Event}, State=#state{}) ->
+    %% Received a sync blockchain_event. Dispatch as if async and then
+    %% acknowledge once it returns
+    Result = handle_info({blockchain_event, Event}, State),
+    blockchain_event:acknowledge(From),
+    Result;
+
+handle_info({blockchain_event, {new_chain, Chain}}, State=#state{follower_mod=FollowerMod}) ->
```
Comment (vihu) [inline src/blockchain_follower.erl:75]: Has the `new_chain` event trumped over the `integrated_genesis_block` event? That seems to be missing from here...

Comment (madninja) [reply src/blockchain_follower.erl:78]: I don't think so (cc @Vagabond ) but block 1 is loaded as part of the catch-up in the next add_block

Comment (Vagabond) [reply src/blockchain_follower.erl:78]: Do we even need `integrated_genesis_block` anymore?

**Comment by madninja** [comment]
Got a verbal 👍 from @Vagabond

**Pull request merged**